src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -96,11 +107,15 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         internal IEnumerable<string> KeyPrefixes => _keyPrefixes;
 
         /// <summary>
-        /// An optional configuration client manager that can be used to provide clients to communicate with Azure App Configuration.
+        /// For use in tests only. An optional configuration client manager that can be used to provide clients to communicate with Azure App Configuration.
         /// </summary>
-        /// <remarks>This property is used only for unit testing.</remarks>
         internal IConfigurationClientManager ClientManager { get; set; }
 
+        /// <summary>
+        /// For use in tests only. An optional class used to process pageable results from Azure App Configuration.
+        /// </summary>
+        internal IPageableConfigurationSettings PageableConfigurationSettings { get; set; }
```

Comment:
I suggested this name (IPageableConfigurationSettings), and I think it was a bad suggestion. Because this is not configuration settings, it is indeed something that iterates through pages. The original name `IConfigurationSettingsPageIterator` would be better.